mud-contracts/world-v2/src/codegen/world/INetworkNodeSystem.sol

```diff
@@ -27,11 +27,11 @@ interface INetworkNodeSystem {
     uint256 currentProduction
   ) external;
 
-  function evefrontier__connectStructure(uint256 networkNodeId, uint256 structureId) external;
+  function evefrontier__connectStructure(uint256 networkNodeId, uint256 assemblyId) external;
 
-  function evefrontier__onStructureOnline(uint256 networkNodeId, uint256 structureId) external;
+  function evefrontier__onStructureOnline(uint256 networkNodeId, uint256 assemblyId) external;
 
-  function evefrontier__onStructureOffline(uint256 networkNodeId, uint256 structureId) external;
+  function evefrontier__onStructureOffline(uint256 networkNodeId, uint256 assemblyId) external;
```

Comment:
do we also want to rename those? 

`evefrontier__connectAssembly`
`evefrontier__onAssemblyOnline`
`evefrontier__onAssemblyOffline`

Reply:
yes on it changing one by one haha